### PR TITLE
edited hover component to tooltip, added flowbite

### PR DESCRIPTION
**Pull request by laeonej**

### CODE REVIEW

File: src/ui/pages/Login/Login.tsx [hunk 1]
```diff
@@ -96,6 +95,7 @@ const Login = () => {
                     </p> 
                 </div >
             </div>
+
```
Comment (Jeffligit) [inline src/ui/pages/Login/Login.tsx:105]: You need to be more careful with adding all files to your commit. these are unnecessary changes because of it.

File: src/ui/components/Hover/Hover.types.tsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+type Placement = 'top' | 'bottom' | 'left' | 'right'
 
 export type HoverProps = {
     icon: IconType;
-    label: string;
     size: number;
-    // textBoxWidth: string;
     color?: string;
+    hoverPosition?: Placement;
     hoverText: React.ReactNode;
```
Comment (Jeffligit) [inline src/ui/components/Hover/Hover.types.tsx]: I suggest renaming hoverText to hoverContent. Text can imply only Text.

**Comment by Jeffligit** [comment]
Resolve conflicts. If you can post snapshots in the pr of the page and component you created. mobile and desktop view.

**Comment by laeonej** [comment]
> Resolve conflicts. If you can post snapshots in the pr of the page and component you created. mobile and desktop view.

Screenshot 2024-08-18 at 7 37 48 PM
Screenshot 2024-08-18 at 7 36 47 PM

**Comment by laeonej** [comment]
Screenshots with the hover component showing content

Screenshot 2024-08-18 at 7 43 08 PM
Screenshot 2024-08-18 at 7 42 44 PM

**Review by Jeffligit — APPROVED** [review]
